src/Test/Utilities/Shared/Compilation/OperationTreeVerifier.cs

```diff
@@ -790,7 +807,8 @@ public override void VisitLiteralExpression(ILiteralExpression operation)
         {
             LogString(nameof(ILiteralExpression));
 
-            if (operation.ConstantValue.HasValue && operation.ConstantValue.Value.ToString() == operation.Text)
+            if (operation.ConstantValue.HasValue &&
+                operation.ConstantValue.Value?.ToString() == operation.Text)
```

Comment:
Dunno if this is correct when the value is null and the text is "null". Maybe null requires a special case here.